INSTALL.md

```diff
@@ -63,9 +63,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: cirruslabs/cirrus-action@v1
+    - uses: cirruslabs/cirrus-action@v2
 ```
 
+**Note:** Cirrus Action integrates natively with GitHub Actions' caching mechanism by using [HTTP Caching Proxy Action](https://github.com/cirruslabs/http-cache-action)
```

Comment:
```
GitHub Actions'
              ^
```

Looks like an [apostrophe](https://en.wikipedia.org/wiki/Apostrophe), but what purpose does it serve here?